delocate/tests/test_delocating.py

```diff
@@ -140,25 +150,43 @@ def test_delocate_tree_libs():
             rp_liba: {rp_slibc: liba,
                       rp_libc: liba,
                       rp_libb: liba}})
-        assert_equal(copied2, exp_dict)
+        assert copied2 == exp_dict
         ext_local_libs = (set(realpath(L) for L in EXT_LIBS) |
                           {liba, libb, libc, slibc})
-        assert_equal(set(os.listdir(copy_dir2)),
-                     set([basename(lib) for lib in ext_local_libs]))
-        # Libraries using the copied libraries now have an install name starting # noqa: E501
+        assert (
+            set(os.listdir(copy_dir2))
+            == {basename(lib) for lib in ext_local_libs}
+        )
+
+
+def test_delocate_path_install_names():
+    # type: () -> None
+    # Test install name modifications.
+    with InTemporaryDirectory() as tmpdir:
+        subtree = pjoin(tmpdir, 'subtree')
+        all_local_libs = _make_libtree(subtree)
+        lib_path = "dynlibs"
+        copied = delocate_path(".", lib_path, copy_filt_func=None)
+        print(copied)
```

Comment:
These are debug prints in tests.  I left them in since they're hidden when tests pass.

I could remove them which would be reasonable since they're pretty vague.